src/status_im/utils/keychain/events.cljs

```diff
@@ -12,6 +13,7 @@
 (re-frame/reg-fx
  :get-encryption-key
  (fn [event]
+   (when platform/desktop? (keychain/set-username))
```

Comment:
Maybe duplicate check for `platform/desktop?` is not necessary.

Reply:
I'm wondering where better to left this check - on higher level in events.cljs or in lower level in `keychain`.  Maybe it's better keep both checks like indicator that the code is expected to be run only on desktop platform (will not cause misunderstanding for mobile devs and for us as well in the future) P.S. I think that piece of code is better than comment.

Reply:
If we have more side-effect of this kind to be triggered eventually when can have a `set-up-desktop-event` and collect them all in one place, but not sure, at the moment not a problem having it here I guess